contracts/addresses/testnet.json

```diff
@@ -17,9 +17,9 @@
   "TAP": "0x6896659267c3c9fd055d764327199a98e571e00d",
   "TOP": "0x2774031b3898fbe414f929b3223ce1039325e7dc",
   "MAKER_OTC": "0x06ef37a95603cb52e2dff4c2b177c84cdb3ce989",
-  "DS_PROXY_FACTORY": "0x0babbba8c5e67472af5c6aaf57bcbe90daa8ddcb",
-  "PROXY_REGISTRY": "0x23f67a19dc232835eaeda2075728f8295f54dfca",
-  "DS_PROXY": "0x570074ccb147ea3de2e23fb038d4d78324278886",
-  "SAI_PROXY": "0x2348a875b1631307577be0935f289e0fb9316169",
-  "OASIS_PROXY": "0xc72b03c37735cf122c27dc352e5f25f75beea389"
+  "DS_PROXY_FACTORY": "0x23f67a19dc232835eaeda2075728f8295f54dfca",
```

Comment:
In the Sai portal and Oasis Direct service, we interact with the proxy factory through the proxy registry (e.g. `build` in the proxy service). Do we need the factory for something new in MCD? If not, I think we can remove this contract.

Reply:
Cool, will do